Documentation/Configuration/TypoScript/Syntax/Conditions/Index.rst

```diff
@@ -0,0 +1,150 @@
+not exist in TSconfig, it is (obviously) impossible to have a backend TSconfig
+condition that checks for a logged in frontend user.
+
+For a reference of allowed condition criteria, please refer to the according
+chapter in the :ref:`frontend TypoScript Reference <t3tsref:conditions>` and
+the :ref:`backend TSconfig Reference <t3tsconfig:conditions>`. These references
+come with examples for single condition criteria as well.
+
+The Tsconfig and TypoScript backend modules show lists of existing conditions
+and allow simulating criteria verdicts to to analyze their impact on the
+resulting TypoScript tree.
+
+Condition criteria are based on
+`symfony expression language <https://symfony.com/doc/current/components/expression_language/syntax.html>`__.
+The core allows extending the symfony expression language with own variables and
+functions, see :ref:`symfony expression language API <sel-within-typoscript-conditions>`
+for more details.
+
+
+Basic example
+=============
+
+.. include:: /CodeSnippets/TypoScriptSyntax/Conditions1.rst.txt
+
+
+.. _typoscript-syntax-conditions-syntax:
+
+Syntax and rules
+================
+
+The general syntax is like this:
+
+.. include:: /CodeSnippets/TypoScriptSyntax/Conditions2.rst.txt
+
+These general rules apply:
+
+* Conditions are encapsulated in :typoscript:`[` and :typoscript:`]`
+
+* :typoscript:`[ELSE]` negates a previous condition criteria and can contain
+  a new body until :typoscript:`[END]` or :typoscript:`[GLOBAL]`. :typoscript:`[ELSE]`
+  is considered if the condition criteria did *not* evaluate to true.
+
+* :typoscript:`[END]` and :typoscript:`[GLOBAL]` stop a given condition scope.
+  This is similar to a closing curly brace :code:`}` in programming languages like PHP.
+
+* .. versionchanged:: 12.0
+
+  :typoscript:`[END]` and :typoscript:`[GLOBAL]` behave exactly the same. Both
+  are kept for historical reasons (for now).
+
+* Conditions can use constants. They are available in frontend TypoScript "setup" and
+  in TSconfig from "site settings. A simple example if this constant
+  :typoscript:`myPageUid = 42` is set:
+
+  .. code-block:: typoscript
+
+      [traverse(page, "uid") == {$myPageUid}]
+          page.10.value = Page uid is 42
+      [end]
+
+* Multiple condition criteria can be combined using :typoscript:`or` or :typoscript:`||`,
+  as well as :typoscript:`and` or :typoscript:`&&`
+
+* Single criteria can be negated using :typoscript:`!`
+
+* Conditions can *not* be nested within code blocks.
+
+* .. versionchanged:: 12.0
+
+  Conditions *can* be nested into each other, if they are located in
+  different snippets (files or records), see example below. They can *not* be nested
+  within the same code snippet.
+
+* A second condition that is *not* :typoscript:`[ELSE]`, :typoscript:`[END]`
+  and :typoscript:`[GLOBAL]` *stops* a previous condition and starts a new one.
+  This is the main reason conditions can *not* be nested within one text snippet.
```

Comment:
how it's related to `.. versionchanged:: 12.0`?  
It has been always (or since long time) like this.

Reply:
The nesting within different files is new I would say

Reply:
11 hat a weird difference between end and global which did not fully work and vanished.

Reply:
Ohh, is there a forge-issue about it?